### PR TITLE
Add `eth_sendRawTransaction` reference page

### PR DESCRIPTION
**Pull request by jlwllmr**
Title.

### CODE REVIEW

File: docs/developers/reference/api/eth-sendrawtransaction.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+The difference lies in the [error responses](#errors). On Linea, `eth_sendRawTransaction` has some 
+unique error responses.
+
+:::
+
+Submits a pre-signed and serialized transaction and returns the transaction hash if successful.
+
+On Linea, this endpoint can be particularly useful for preventing transactions from failing as a 
+result of exceeding the sequencer's line limit per block. An error message is returned if the line
+count check fails, indicating that the transaction will fail.
```
Comment (bgravenorst) [inline docs/developers/reference/api/eth-sendrawtransaction.mdx:23]:
```suggestion
On Linea, this endpoint proves useful in preventing transaction failures due to exceeding the sequencer's per-block line limit. If the line count check fails, it returns an error message indicating the impending transaction failure.
```

File: docs/developers/reference/api/eth-sendrawtransaction.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+The line limit is a threshold for how much transaction data can effectively be added to a single
+proof; too many lines of data, and the prover is unable to add the transaction to the proof within
+the time limit. A line count check is therefore imposed on every transaction to ensure 
+transactions are successful, and able to be added to a proof.
```
Comment (bgravenorst) [inline docs/developers/reference/api/eth-sendrawtransaction.mdx:28]:
```suggestion
The line limit is a threshold for the amount of transaction data that can be added to a single
proof. Exceeding this limit means the prover cannot include the transaction in the proof within
the time limit. Therefore, every transaction undergoes a line count check to ensure its
success and inclusion in a proof.
```

**Review by bgravenorst — COMMENTED** [review]
Left some minor editing comments.

However, should we include a "JSON RPC API" section (or similar) to the "Differences between Ethereum and Linea" topic as well?

We can link off to this content.

**Comment by jlwllmr** [comment]
@bgravenorst thanks for the suggestions. 

> However, should we include a "JSON RPC API" section (or similar) to the "Differences between Ethereum and Linea" topic as well?

I think this makes sense. I'll make an issue!

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**